app.py

```diff
@@ -121,6 +134,35 @@ def _print_version(ctx, _, value):
     ctx.exit()
 
 
+def _send_metrics():
+    """Send metrics to pushgateway."""
+    pushgateway_url = THOTH_METRICS_PUSHGATEWAY_URL
+    deployment_name = THOTH_DEPLOYMENT_NAME
+
+    component_name = "revsolver"
```

Comment:
`THOTH_METRICS_PUSHGATEWAY_URL`, `THOTH_DEPLOYMENT_NAME` and `component_name` can be directly passed to the `push_to_gateway` call.